website/docs/recipes/munit.md

```diff
@@ -0,0 +1,115 @@
+---
+id: munit
+title: MUnit (Mule4) with Pact
+description: How to use Pact for testing MuleSoft (Mule 4) APIs with MUnit
+---
+
+# Problem Statement: Pact with Mule 4 (MuleSoft)
+
+As of Mule 4, it is no longer possible to extend the MUnit runner through Java code. Previously, the solution of Pact testing with MuleSoft was to extend “FunctionalMunitSuite” in your test class. An example can be found here: https://docs.mulesoft.com/munit/1.3/munit-tests-with-Java#creating-your-suite-class
+
+However, MuleSoft now wants alignment with the Mule language to be able to provide a better service. Which leaves the outstanding problem; How can we use Pact to test Mule 4 applications? 
+
+In a nutshell, the solution is to have MUnit be the test runner and execute the standalone pact-JVM server by calling Java code within the MUnit console. This will allow for both the benefit of MUnit tests along with pact files being created for contract testing. As a result, we will be able to leverage the MUnit unit test coverage while also generating our pact files and publishing them to a Pact Broker or sharing them into your own file storage system. 
+
+# Consumer
```

Comment:
same comments as the below one about headings, this should be ## in order to show up in the nav bar. you will need to adjust the existing ## level ones to ###